config/webpack/test.js

```diff
@@ -12,8 +12,16 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'test'
 process.env.RAILS_ENV = process.env.RAILS_ENV || 'test'
 
 const environment = require('./environment')
+const nodeExternals = require('webpack-node-externals');
 
-const babelLoader = environment.loaders.get('babel')
-babelLoader.include.push(babelLoader.include[0].concat('../../test'))
```

Comment:
This doesn't seem to be necessary anymore 🤷 
I assume https://github.com/maebeale/mutual-aid/pull/71 made the difference